### PR TITLE
AB#62111 added silent & changed names

### PR DESCRIPTION
**Pull request by lardo-de-arnaud**
edited README, made curl silent and improved the name of the script & folder

### CODE REVIEW

**Comment by lardo-de-arnaud** [comment]
This is the PR where I accidentally committed beta.env

**Comment by nicholaswilson100** [comment]
So to remove line 7 in beta. env you just need to do 'git checkout [AB#62111](https://dev.azure.com/oxforduniversity/a2417215-bc45-4732-9067-4006e8242f57/_workitems/edit/62111)' then remove the line (or replace it with whatever you want) then 'git add beta.env ; git commit -m "Removing access token" beta.env ; git push origin [AB#62111](https://dev.azure.com/oxforduniversity/a2417215-bc45-4732-9067-4006e8242f57/_workitems/edit/62111)'

File: setup-azad-auth/beta.env [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# The hostname to update
+#host=universityofoxford.test.instructure.com
+host=universityofoxford.beta.instructure.com
+#host=universityofoxford.instructure.com
+
+# Bearer token
+token="12201~TEF1MhBoJMQrYxi2EbUdQAdCMCfNhOJ7du6ICwkc9o5nnIUiQbwKMdzPKQj5Yriu"
```
Comment (nicholaswilson100) [inline setup-azad-auth/beta.env]: Is this token supposed to be in github? PRs I think are publicly viewable things

Comment (nicholaswilson100) [reply setup-azad-auth/beta.env]: The rest looks fine to me

**Pull request merged**